### PR TITLE
BaSP-TG-08: Task table - Resources list

### PR DESCRIPTION
**Pull request by TomasBettini**
**Show a table with the list of resources of the assigned entity, saved in the database.
For each row, you must be able to:**
- *Edit an item.*
- _Delete an item._
- _Show success and error messages._

**For the whole table:**
- Create a redirect button to 'Add feature'

### CODE REVIEW

File: src/Components/Tasks/Form/index.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  const [modalDisplay, setModalDisplay] = useState('');
+  const [contentMessage, setContentMessage] = useState('');
+  const [modalTitle, setModalTitle] = useState('');
+
+  const editAndCreateMessage = (contentSubTitle, description) => {
+    return `${contentSubTitle}:\n
+     Description: ${description}`;
+  };
+
+  if (idRegEx.test(urlID)) {
```
Comment (Juancapp) [inline src/Components/Tasks/Form/index.js:22]: Put the IF conditional into the UseEffect

**Review by Juancapp — DISMISSED** [review]
Nice code!!!

**Review by Juancapp — DISMISSED** [review]
Nice code!!!

**Review by juliancit0 — DISMISSED** [review]
Good job!

File: src/Components/Tasks/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            <input
+              id="input-name"
+              name="description"
+              required
+              value={descriptionValue}
+              onChange={changeDescription}
+            />
+          </div>
+          <div>
+            <a href={'http://localhost:3000/tasks'}>
```
Comment (leimeter-joaquin) [inline src/Components/Tasks/Form/index.js:98]: don't link to localhost use the env variable

**Review by leimeter-joaquin — CHANGES_REQUESTED** [review]

**Review by leimeter-joaquin — APPROVED** [review]

**Review by Alfondalix — APPROVED** [review]

**Comment by disimoneleandro** [comment]
QA pased!

**Pull request merged**